fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java

```diff
@@ -53,6 +54,10 @@
       "X-RateLimit-Reset"
   );
 
+  private static String GRAPHQL_QUERY_TPL = "{\norganization(login: \"ORGANIZATION\") {\nteams(first: 100, userLogins: [\"USER\"]) {\nedges {\nnode {\nid\nname\nslug\n}\n}\n}\n}\n}\n\"";
```

Comment:
Do these need the new line characters in the query? If not, it would be much easier to read without them. If they're required, I suggest doing something like:

```java
String.join("\n",
"{",
"organization(login): ...",
"teams(first: 100, ...)",
"}");